evento/app/controllers/users_controller.rb

```diff
@@ -63,7 +64,11 @@ def set_user
   end
 
   # Only allow a trusted parameter "white list" through.
-  def user_params
-    params.require(:user).permit(:name, :email, :password)
+  def create_params
```

Comment:
`put "/users/#{user.id}", params: { "name": "NewName"}.to_json, headers: { "Authorization": json["auth_token"], 'CONTENT_TYPE' => 'application/json' }`

can be simplified to:
`let(:header)  { { Authorization: json["auth_token"], CONTENT_TYPE: "application/json" } }`
and 
`put "/users/#{user.id}", params: { name: "NewName"}.to_json, headers: header`